Sources/PointFree/Episode/Show.swift

```diff
@@ -34,7 +34,6 @@ let episodeResponse =
           currentUser: currentUser,
           data: (permission, currentUser, subscriberState, episode),
           description: episode.blurb,
-          extraHead: videoJsHead,
```

Comment:
This'll temporarily making video.js videos less featured, right?

Reply:
oh yes, I forgot we still have some video.js. I'll bring it back for now